src/test/common/components/payment-intention/model/paymentPlan.ts

```diff
@@ -14,6 +14,7 @@ const DEFAULT_PAYMENT_PLAN = {
   instalmentAmount: 50,
   firstPaymentDate: { year: FUTURE_YEAR, month: 10, day: 10 },
   paymentSchedule: PaymentSchedule.EVERY_MONTH.value,
+  completionDate: MomentFactory.parse('2028-10-10'),
```

Comment:
Here you are only allowed to have primitive types. Remember the payload you are constructing is meant to simulate a form object sent by the browser.